### ISSUE TITLE
Trouble with gEquals types

### ISSUE THREAD

**Issue by josh-s-g** [tminglei/slick-pg#443]
Using `gEquals` for geometry equality comparisons: 
```scala
  def foo(toFind: Option[Geometry]): DBIO[Seq[Int]] = {
    toFind match {
      case Some(f) => someTable.filter(_.geometry.gEquals(f.bind)).map(_.id).result
      case None    => DBIO.successful(Seq.empty)
    }
  }
```
Getting an error with the types of `ST_Equals`:

```
org.postgresql.util.PSQLException: ERROR: function st_equals("someSchema".geometry, bytea) does not exist
  Hint: No function matches the given name and argument types. You might need to add explicit type casts.
```
What is that `bytea` doing? Or is this likely something to be related to the schema that PostGIS is installed onto?

Thanks

**Comment by tminglei** [comment]
@josh-s-g did you install postgis extension?

_After installed the software, you still need enable it by executing `CREATE EXTENSION IF NOT EXISTS postgis` in the target database._

**Comment by josh-s-g** [comment]
Yes, I've been using plenty of other postgis and slick-pg functions - it's just `gEquals` that I'm having trouble with.

Thanks

**Comment by josh-s-g** [comment]
I think it might be a schema-wide problem; I've had the same issue with other slick-pg functions (such as 
`org.postgresql.util.PSQLException: ERROR: function st_dwithin("someSchema".geometry, bytea, numeric) does not exist`

**Comment by josh-s-g** [comment]
So PostGIS was not installed to the `public` schema. Can slick-pg support this case?

**Comment by tminglei** [comment]
No. You should install postgis by youself.
